core/src/test/java/org/bitcoinj/script/ScriptTest.java

```diff
@@ -226,32 +226,65 @@ public void createAndUpdateEmptyInputScript() throws Exception {
     public void testOp0() {
         // Check that OP_0 doesn't NPE and pushes an empty stack frame.
         Transaction tx = new Transaction(PARAMS);
-        tx.addInput(new TransactionInput(PARAMS, tx, new byte[] {}));
+        tx.addInput(new TransactionInput(PARAMS, tx, new byte[]{}));
         Script script = new ScriptBuilder().smallNum(0).build();
 
         LinkedList<byte[]> stack = new LinkedList<byte[]>();
         Script.executeScript(tx, 0, script, stack, Script.ALL_VERIFY_FLAGS);
         assertEquals("OP_0 push length", 0, stack.get(0).length);
     }
 
+    static final int MAX_BITWISE_RANDOM_TESTS = 2000;
+
+    @Test
+    public void testBitwiseOneByteLength() throws IOException {
```

Comment:
This test only check one single byte operations, but multiple bytes operations can be created also.